spec/jobs/reports/monthly_key_metrics_report_spec.rb

```diff
@@ -80,23 +82,22 @@
   it 'uploads a file to S3 based on the report date' do
     expect(subject).to receive(:upload_file_to_s3_bucket).with(
       path: account_reuse_s3_path,
-      body: anything,
-      content_type: 'text/csv',
-      bucket: 'reports-bucket.1234-us-west-1',
+      **s3_metadata,
     ).exactly(1).time.and_call_original
 
     expect(subject).to receive(:upload_file_to_s3_bucket).with(
       path: total_profiles_s3_path,
-      body: anything,
-      content_type: 'text/csv',
-      bucket: 'reports-bucket.1234-us-west-1',
+      **s3_metadata,
     ).exactly(1).time.and_call_original
 
     expect(subject).to receive(:upload_file_to_s3_bucket).with(
       path: account_deletion_rate_s3_path,
-      body: anything,
-      content_type: 'text/csv',
-      bucket: 'reports-bucket.1234-us-west-1',
+      **s3_metadata,
```

Comment:
I thought I'd try to DRY this up a little rather than copying it for a 4th time.